devito/types/dense.py

```diff
@@ -981,6 +987,19 @@ def space_order(self):
         """The space order."""
         return self._space_order
 
+    @property
+    def coefficients(self):
+        """Form of the FD coefficients of the function."""
+        return self._coefficients
+
+    @cached_property
+    def coeff_symbol(self):
```

Comment:
private method

Reply:
This property has now been removed and replaced with a function in `coefficients.py`.

Reply:
Actually, after thinking about future functionality, I think it may be better to keep this symbol as a property of the function it is attached to. The construction of the symbol has now been made private.